lib/tasks/dev.rake

```diff
@@ -1,3 +1,30 @@
+namespace :marc_liberation do
+  namespace :server do
```

Comment:
Definitely seems like it belongs in `dev.rake`. I created a follow-up ticket: https://github.com/pulibrary/marc_liberation/issues/1025